### ISSUE TITLE
Calibration makes surround view stitching more distorted

### ISSUE THREAD

**Issue by Tamme** [intel/libxcam#657]
Hi.

I calibrated my cams using matlab. Initially the projection is totally messed up:
![image](https://user-images.githubusercontent.com/3532047/62302380-ead3ad80-b482-11e9-9a45-0ddfa31f76cc.png)

After i change some signs of values in the intrinsic calibration inverse mapping it becomes normal, but the fisheye effect is actually increased:
![image](https://user-images.githubusercontent.com/3532047/62302447-0b9c0300-b483-11e9-9845-b1d1d28482fd.png)

After tinkering with pitch (set it to 15 degs) I can get it to be less distorted, bus my cameras are definitely not at this angle. 
![image](https://user-images.githubusercontent.com/3532047/62302541-36865700-b483-11e9-830f-05f99391a7c6.png)

Did you maybe use some other calibration setup, or have other ideas, what could be messed up?

Thank you!

**Comment by mrp1234567** [comment]
@Tamme need some help on surround view with libxcam, can you please list the steps you did ?

**Comment by Tamme** [comment]
Hi @mrp1234567. As mentioned, I used matlab ocam calib toolbox to get calibration values for the intrinsics. It has been some time, but I think then used [this example](https://github.com/intel/libxcam/blob/master/tests/test-image-stitching.cpp) as the executable to run the stitcher. And then I got some reasonable results with all zero extrinsics, I started modifying them, to make the result more correct.

Hope it helps.
